.pipelines/onebranch.buddy.yml

```diff
@@ -19,7 +19,7 @@ resources:
       ref: refs/heads/main
 
   pipelines:
-    - pipeline: vscode-website-main
```

Comment:
I'm not sure I understand the point of this rename. the pipeline name is a nickname for the pipeline that we ingest. we are ingesting the vscode-website/main and vscode-website/release/prod so the names match their contents today.

Reply:
vscode-website-main is fine as a name, but there is another pipeline named vscode-website-prod already in the website repo. I noticed that the website buddy build triggers the docs buddy build, but that the website official build doesn't trigger the docs official build.